ui/.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-
+yarn.lock 
```

Comment:
If we're no longer using Yarn, why ignore its lockfile?

Reply:
just there for dev mode

Reply:
Which 'dev mode'? Either everyone uses Yarn/`yarn.lock`, or everyone uses NPM/`package-lock.json`. We can't have two systems in use at the same time.

Reply:
you can use yarn or npm when you develop to install your dependencies. If you use yarn, il prevent you to commit the yarn.lock. that's it.

Reply:
Seems fair to protect the repo from unexpected committed files from the two main tools the web developers could use.

Reply:
The point is a web developer should not be allowed to 'pick one of the two'. We, the project, pick one. Otherwise you end up with half of your devs using version A of the deps (the ones using NPM, and a checked-in `package-lock.json`), and half of the devs using version X, Y or Z (the ones using Yarn, and no `yarn.lock` in the repo).